test/TestServerProjectsLowLevel/dpg-customization/Generated/DPGClientOptions.cs

```diff
@@ -27,6 +27,7 @@ public enum ServiceVersion
         /// <summary> Initializes new instance of DPGClientOptions. </summary>
         public DPGClientOptions(ServiceVersion version = LatestVersion)
         {
+            Diagnostics.SuppressNestedClientSpans = true;
```

Comment:
We will set the SuppressNestedClientSpans true to suppressNestedClientSpans, but I saw the ClientDiagnostics constructor in ClientDiagnostics.cs. The Diagnostics.SuppressNestedClientSpans is not used at all. It is hard-set false.  Is there any mismatch between Azure.Core and autorest.csharp?


![image](https://user-images.githubusercontent.com/10458143/165922500-36273b26-a498-4836-8b8e-4635a7ca1856.png)

Reply:
As I mentioned [here](https://github.com/Azure/autorest.csharp/pull/2184#issuecomment-1107117271), we are tracking this issue [here](https://github.com/Azure/azure-sdk-for-net/issues/28354). We also have a [test case](https://github.com/Azure/autorest.csharp/blob/308f082d8f98f707a11f781021068b9524f57c97/test/AutoRest.TestServerLowLevel.Tests/dpg-customization.cs#L32) to check that the nested spans are being suppressed are not and currently it is failing because of this issue.

Reply:
As I mentioned [here](https://github.com/Azure/autorest.csharp/pull/2184#issuecomment-1107117271), this PR is blocked and we are tracking this issue [here](https://github.com/Azure/azure-sdk-for-net/issues/28354). We also have a [test case](https://github.com/Azure/autorest.csharp/blob/308f082d8f98f707a11f781021068b9524f57c97/test/AutoRest.TestServerLowLevel.Tests/dpg-customization.cs#L32) to check that the nested spans are being suppressed are not and currently it is failing because of this issue.